script/configs/mainnet/v0.3.0-mainnet-rewards.config.json

```diff
@@ -0,0 +1,57 @@
+{
+  "chainInfo": {
+    "chainId": 1
+  },
+  "multisig_addresses": {
+    "communityMultisig": "0xFEA47018D632A77bA579846c840d5706705Dc598",
+    "executorMultisig": "0x369e6F597e22EaB55fFb173C6d9cD234BD699111",
+    "operationsMultisig": "0xBE1685C81aA44FF9FB319dD389addd9374383e90",
+    "pauserMultisig": "0x5050389572f2d220ad927CcbeA0D406831012390",
+    "timelock": "0xA6Db1A8C5a981d1536266D2a393c5F8dDb210EAF"
+  },
+  "strategies": {
+    "numStrategies": 0,
+    "MAX_PER_DEPOSIT": 115792089237316195423570985008687907853269984665640564039457584007913129639935,
+    "MAX_TOTAL_DEPOSITS": 115792089237316195423570985008687907853269984665640564039457584007913129639935,
+    "strategiesToDeploy": []
+  },
+  "strategyManager": {
+    "init_strategy_whitelister": "0xBE1685C81aA44FF9FB319dD389addd9374383e90",
+    "init_paused_status": 0
```

Comment:
would probably be good if we can clean these up by deleting irrelevant (to this deployment) parts of the config. I understand it not being worth it if deleting entries breaks other things though!